x-pack/plugins/secops/public/components/page/hosts/host_summary/index.tsx

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  EuiDescriptionList,
+  EuiDescriptionListTitle,
+  EuiFlexItem,
+  EuiHorizontalRule,
+  EuiPanel,
+  EuiTitle,
+  EuiToolTip,
+} from '@elastic/eui';
+import { FormattedRelative } from '@kbn/i18n/react';
+import { get, getOr, isArray } from 'lodash/fp';
+import React from 'react';
+import { pure } from 'recompose';
+import { HostItem, HostsEdges } from 'x-pack/plugins/secops/server/graphql/types';
+import { DragEffects, DraggableWrapper } from '../../../drag_and_drop/draggable_wrapper';
+import { escapeDataProviderId } from '../../../drag_and_drop/helpers';
+import { getEmptyValue } from '../../../empty_value';
+import { Provider } from '../../../timeline/data_providers/provider';
+import * as i18n from './translations';
+
+interface OwnProps {
+  data: HostsEdges[];
+  loading: boolean;
+  startDate: number;
+  endDate: number;
+}
+
+type HostSummaryProps = OwnProps;
+
+export const HostSummary = pure<HostSummaryProps>(({ data, startDate, endDate, loading }) => (
+  <EuiFlexItem style={{ maxWidth: 750 }}>
+    <EuiPanel>
+      <EuiTitle size="s">
+        <h3>{i18n.SUMMARY}</h3>
+      </EuiTitle>
+
+      <EuiHorizontalRule margin="xs" />
+      {getEuiDescriptionList(getOr(null, 'node', data[0]), startDate, endDate)}
+    </EuiPanel>
+  </EuiFlexItem>
+));
+
+const fieldTitleMapping: Readonly<Record<string, string>> = {
+  'host.name': i18n.NAME,
+  firstSeen: i18n.LAST_BEAT,
+  'host.id': i18n.ID,
+  'host.ip': i18n.IP_ADDRESS,
+  'host.mac': i18n.MAC_ADDRESS,
+  'host.type': i18n.TYPE,
+  'host.os.platform': i18n.PLATFORM,
+  'host.os.name': i18n.OS_NAME,
+  'host.os.family': i18n.FAMILY,
+  'host.os.version': i18n.VERSION,
+  'host.architecture': i18n.ARCHITECTURE,
+};
```

Comment:
If I had a 💵for how many times I have seen one of these mappers in a UI project :-). I like this, this is an ok pattern. Sometimes they grow very large and end up in another file. This is good here right now.

Reply:
Ha! It's a fan favorite for this sorta stuff! I could see this component maturing into a general purpose key/value display, so I had that in mind while writing it. It gets fun (and complicated as you mentioned) when you try to jam more and more into the descriptor config (e.g. handling multiple field types, display preferences, different controls, etc), but we'll keep an eye on it to make sure it grows responsibly :)